packages/mobile/src/validation-schemas/user/sign-up.ts

```diff
@@ -1,21 +1,158 @@
 import * as Joi from 'joi';
-import { UserValidationMessage, UserSignUpDto } from '@vse-bude/shared';
+import { UserSignUpDto } from '@vse-bude/shared';
+import i18next from 'i18next';
+
+const emailValidator = (value: string, helpers: Joi.CustomHelpers) => {
```

Comment:
I'd suggest extracting all these validation helpers either to `helpers/validation` or to `validation-schemas/helpers` as you might need some of those to be used in other schemas in future

Reply:
corrected